conf/gconf/cluster.conf.sample

```diff
@@ -0,0 +1,357 @@
+      - https://csm.seagate.com:8081
+    auth_admin: admin                               # deprecated
+    auth_secret: !!binary |                         # deprecated
+      UTdTaDl0TzVLMWZsQUp6SnVKeTZaYlNtUUFscld1WjU4UHNSek1aUGc9PQ==
+    email_address: cortx@seagate.com # Optional 
+    mgmt_admin: cortxadmin
+    mgmt_secret: !!binary |
+      UlU1U0NrYklraS0ta3BOamN5X2J1VmUwRnJXTUFlb2JDdkItY1B3ZEE9PQ==
+    limits:  
+      num_services: 1  
+      services:              
+      - name: agent
+        memory:
+          min: 128Mi
+          max: 256Mi
+        cpu:
+          min: 250m
+          max: 500m
+  ha:
+    limits:         
+      num_services: 1 
+      services:        
+      - name: fault_tolerance
+        memory:
+          min: 128Mi
+          max: 1Gi
+        cpu:
+          min: 250m
+          max: 500m
+      - name: health_monitor
+        memory:
+          min: 128Mi
+          max: 1Gi
+        cpu:
+          min: 250m
+          max: 500m
+      - name: k8s_monitor
+        memory:
+          min: 128Mi
+          max: 1Gi
+        cpu:
+          min: 250m
+          max: 500m
+    health:                                     # TBD: Future. Do not use for now
+      num_endpoints: 1
+      endpoints:
+      - http://server1-node1.com:22000
+    
+  s3:
+    iam:                                        # To be removed
+      endpoints:
+      - https://cortx-io-svc:9443
+      - http://cortx-io-svc:9080
+    data:
+      endpoints:
+      - http://cortx-io-svc:80
+      - https://cortx-io-svc:443
+    internal:                                   # To be removed
+      endpoints:
+      - http://cortx-io-svc:28049
+    io_max_units: 8
+    max_start_timeout: 240
+    auth_user: user_name                        # (new) IAM user name 
+    auth_admin: sgiamadmin
+    auth_secret: s3_auth_admin_secret
+    service_instances: 2                        # deprecated
+    limits:                  
+      num_services: 1
+      services:
+      - name: s3
+        memory:
+          min: 128Mi
+          max: 2Gi
+        cpu:
+          min: 250m
+          max: 1000m
+  hare:
+    hax:
+      num_endpoints: 10
+      endpoints:
+      - https://motr-hax-svc:22003              # Control endpoint
+      - tcp://data1-node1:22001                 # For motr and Hax communication
+      - tcp://data1-node2:22001                 # For motr and Hax communication
+      - tcp://data1-node3:22001                 # For motr and Hax communication
+      - tcp://server-node1:22001                # if s3 client is configured
+      - tcp://server-node2:22001                # if s3 client is configured
+      - tcp://server-node3:22001                # if s3 client is configured
+      - tcp://client-node1:22001               # Only if motr_client is configured
+      - tcp://client-node2:22001               # Only if motr_client is configured
+      - tcp://client-node3:22001               # Only if motr_client is configured

```

Comment:
3 new endpoints has been added in page : https://seagate-systems.atlassian.net/wiki/spaces/PUB/pages/611615649/CORTX+Configuration+Parameters  ,do add them here also.

Reply:
done